Sources/PT.PM.Common/MultiMap.cs

```diff
@@ -53,6 +53,16 @@ public void Add(TKey key, TValue value)
             storageValue.Add(value);
         }
 
+        public void Add(TKey key, IEnumerable<TValue> values)
```

Comment:
Maybe rename it `AddRange` for consistency with other collections? But I'm not 100% sure about this one...

Reply:
I think it's ok because of it's `MultiMap` but not `Dictionary`.